startgoofcord

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Set FEATURES to what goes after --enable-features; for example, UseSkiaRenderer,WebRTCPipeWireCapturer,etc,etc will then get turned into --enable-features=UseSkiaRenderer,WebRTCPipeWireCapturer,etc,etc.
+FEATURES="UseSkiaRenderer,VaapiVideoDecoder,VaapiVideoEncoder,PulseaudioLoopbackForScreenShare,VaapiVideoDecodeLinuxGL"
+export TMPDIR="$XDG_RUNTIME_DIR/app/${FLATPAK_ID:-io.github.milkshiift.GoofCord}"
+declare -a FLAGS=(--ozone-platform-hint=auto --enable-gpu-rasterization --enable-zero-copy --enable-gpu-compositing --enable-native-gpu-memory-buffers --enable-oop-rasterization --disable-gpu-blocklist)
+
+if [[ $XDG_SESSION_TYPE == "wayland" ]]
```

Comment:
These should check for the wayland socket. XDG_SESSION_TYPE is not enough because someone can use `--nosocket=wayland` while being on a wayland session.

environment variables are inherted from the host and has no connection with finish args.